lib/MusicBrainz/Server/Controller/Admin/Attributes.pm

```diff
@@ -115,10 +117,28 @@ sub delete : Chained('attribute_base') Args(1) RequireAuth(account_admin) Secure
     $c->stash->{attribute} = $attr;
 
     if ($c->model($model)->in_use($id)) {
-        $c->stash->{template} = 'admin/attributes/in_use.tt';
+        my $error_message = l('You cannot remove the attribute "{name}" because it is still in use.', { name => $attr->name });
+
+        $c->stash(
+            current_view => 'Node',
+            component_path => 'admin/attributes/CannotRemoveAttribute',
+            component_props => {message => $error_message}
+        );
+        
         $c->detach;
     }
 
+    if ($c->model($model)->has_children($id)) {
+        my $error_message = l('You cannot remove the attribute "{name}" because it is the parent of other attributes.', { name => $attr->name });
```

Comment:
Can use typographic quotes when introducing new message:
```suggestion
        my $error_message = l('You cannot remove the attribute “{name}” because it is the parent of other attributes.', { name => $attr->name });
```